connect/runtime/src/main/java/org/apache/kafka/connect/runtime/ConnectorConfig.java

```diff
@@ -43,7 +43,9 @@
 
     public static final String CONNECTOR_CLASS_CONFIG = "connector.class";
     private static final String CONNECTOR_CLASS_DOC =
-            "Name of the class for this connector. Must be a subclass of org.apache.kafka.connect.connector.Connector";
+                    "Name or alias of the class for this connector. Must be a subclass of org.apache.kafka.connect.connector.Connector. " +
+                    "If the connector is org.apache.kafka.connect.file.FileStreamSinkConnector, you can either specify this full name, " +
+                    " or use \"FileStreamSink\" or \"FileStreamSinkConnect\" to make the configuration a bit shorter";
```

Comment:
I think you mean `\"FileStreamSinkConnector\"` here.

Also, I know it doesn't work for the `FileStream` case, but should we support also dropping the `Sink` or `Source` part? I don't feel like I have a good feel for how frequently we'll have both implemented. For example, while you could implement both source and sink for JDBC, I'd guess that the vast majority of users are looking for the source connector and so that is likely the only JDBC connector class available.